src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -251,18 +256,21 @@ await foreach (var entry in _hashCache.IndexFolderAsync(location))
         }
 
         // Write the generated files (could be done in parallel)
+        if (toWrite.Count > 0 && (loadout == null || flattenedLoadout == null))
```

Comment:
Note that changes to this method could be reverted if requested. It's no longer necessary as `FastApplyOriginalGameState` now just uses the marker loadout rather than directly applying the original tree.